README.md

```diff
@@ -37,3 +37,9 @@ In order to run the gui:
 You may need to specify the gem5 root directory if the above yields import errors. On the command line:
 
     GEM5_HOME=<path to gem5>
+    
+## Importing objects from config scripts
+You can import custom made simobjects from python scripts into the gui. However before doing so you need to include the following code snippet in your script. 
+
+    import sys, os
+    sys.path.append(os.getenv('gem5_path'))
```

Comment:
is there a way to do this weithout having to add these lines? it would be a hassle to go through all .py files and add these lines

Reply:
I don't remember why I added those lines but you actually don't need to do that lol.